### ISSUE TITLE
Negative ion display loss in the simplified notation of carboxyl negative ions

### ISSUE THREAD

**Issue by OldSuns** [Acylation/obsidian-chem#52]
### What happened?

My Smiles Line:
[NH3+]C(CS)C([O-])=O-
Negative ion display loss in the simplified notation of carboxyl negative ions
![image](https://github.com/Acylation/obsidian-chem/assets/34978144/885b058c-2417-4cc7-99ae-ac6a8e45840b)

### What did you expect to happen?

Negative ion should be displayed.

### How can we reproduce it (as minimally and precisely as possible)?

Update it

### Plugin version

0.3.0

### Obsidian version

1.5.3

**Comment by Acylation** [comment]
Yeah that's why we need the option `Compact drawing` [Sigh]. For a temporarily workaround please disable this option.

This happens upstream and it requires too much work for me to fix it. I'm about to include more render cores to bypass the problem. And, a per-structure config override is in design, you would be able to just disable the `compact drawing` option for only this smiles.

Thanks for your report and any following ideas are warm welcomed.

**Comment by antanyazov** [comment]
Now in Smiles Drawer there aren`t any negative ion at all in some cases even if compact drawing is disabled. This smiles  `C[C-]CO` are displayed without any charge. Yes, RDKit works good with it but it doesn`t support some pretty big molecules I really want. 
![image](https://github.com/user-attachments/assets/631320a9-72de-4727-be88-0ba0587995f8)
